js/plugins/dotprompt/src/registry.ts

```diff
@@ -80,7 +80,7 @@ export async function loadPromptFolder(
         promptsPath,
         {
           withFileTypes: true,
-          recursive: false,
+          recursive: true,
```

Comment:
how would this handle:
```
prompts
  |- foo.prompt
  |- subfolder
    |- foo.prompt
```

Reply:
Good call, let me update the namespace to include the sub-directory path (so this would be registered as `dotprompt/subfolder/foo`)

Reply:
Done.